src/rulesets/reader.ts

```diff
@@ -1,34 +1,46 @@
 import { merge } from 'lodash';
-import Lint from '../cli/commands/lint';
 import { readParsable } from '../fs/reader';
 import { RuleCollection } from '../types';
 import { IRulesetFile } from '../types/ruleset';
 import { formatAjv } from './ajv';
 import { resolvePath } from './path';
 import { validateRuleset } from './validation';
 
-export async function readRulesets(command: Lint, ...files: string[]): Promise<RuleCollection> {
-  const rulesets = await Promise.all(files.map(file => readRuleset(command, file)));
+export async function readRulesFromRulesets(...files: string[]): Promise<RuleCollection> {
```

Comment:
Is this a more appropriate name? It is reading the full contents of the ruleset file, not just the rules section. It's reading the extend, and any other information we add to it.

Reply:
But it returns rules, not the whole ruleset, so yes it's more appropriate imo.